backend/danswer/chat/process_message.py

```diff
@@ -745,6 +745,13 @@ def stream_chat_message_objects(
                     tool_result = packet
                 yield cast(ChatPacket, packet)
         logger.debug("Reached end of stream")
+    except ValueError as e:
+        error_msg = str(e)
+        logger.exception(f"Failed to process chat message: {error_msg}")
```

Comment:
I believe logger.exception automatically logs the exception, so the log line for this will double print. https://www.loggly.com/blog/exceptional-logging-of-exceptions-in-python/

Reply:
Great point!